docs/DesignDocs/InitTimeConfigurationInterface.md

```diff
@@ -83,28 +83,66 @@ typedef enum _oe_enclave_setting_type
      *  reflected in TEE-produced enclave identity evidence. Currently only
      *  supported by SGX Enclaves with KSS feature enabled.
      */
-    OE_SGX_CONFIGURATION_DATA = 0x78b5b41d
+    OE_SGX_ENCLAVE_CONFIG_DATA = 0x78b5b41d
```

Comment:
OE guidance is to avoid abbreviations (like "CONFIG").  The old name had CONFIGURATION in it